docs/UserGuide.md

```diff
@@ -282,9 +285,21 @@ You can use `list` or `find` (e.g `find n/Amy`) to display her information again
 
 </div>
 
+<div markdown="block" class="alert alert-danger">
+
+:warning: You cannot add additional details with this command.
+
+</div>
+
 Examples:
-*  `edit 1 n/Alex p/99999999 s/Meridian Junior College` Edits the name, phone number and school of the 1st student to be `Alex`, `99999999` and `Meridian Junior College` respectively.
-*  `edit 3 v/Blk 33 West Coast Rd #21-214 t/1 1430-1630` Edits the venue and time of the third student to be `Blk 33 West Coast Rd #21-214` and `1 1430-1630` respectively.
+*  `edit 1 n/Arthur p/98727030 s/Meridian Junior College` edits the name, phone number and school of the 1st student to be `Arthur`, `98727030` and `Meridian Junior College` respectively.
+
+Expected Outcome:
+The following figure (Figure __) shows the expected outcome after executing the above example.
```

Comment:
We are just gonna go with "the figure below shows..." instead of explicitly stating the figure number, so I think you can leave out the parenthesis

Reply:
Oh wow nvm

Reply:
well, i didnt see this comment. Anyway i think for CS2101 we need to have it in there, so its fine